### PR TITLE
Flag Path Update to remove "//"

### LINKED ISSUE #616: Flags not displayed in UI on Windows systems due to "//" in path
There are multiple spots in the UI where the flag png is not found due to "//" being used in the path.  On Linux systems the "//" is ignored however on Windows this is an error and the png is not loaded.

Example: 
[12/Sep/2023 14:52:09] "GET /static//img/gcd/flags/fr.png HTTP/1.1" 500 59

### PR DESCRIPTION
**Pull request by prutledge**
Fix #616 - update creation of path url to remove double /

### CODE REVIEW

**Comment by LegalizeAdulthood** [comment]
LGTM

File: apps/indexer/templates/indexer/bits/profile_display.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <li> Show links to the wiki documentation: {{ profile_user.indexer.show_wiki_links|yesno }}
   <li> Use tabbed view while editing stories: {{ profile_user.indexer.use_tabs|yesno }}
 {% endif %}
-  <li> Country: <img src="{{ STATIC_URL }}/img/gcd/flags/{{ profile_user.indexer.country.code|lower }}.png" alt="{{ country }}" class="embedded_flag">
```
Comment (jochengcd) [inline apps/indexer/templates/indexer/bits/profile_display.html:17]: You do have a working dev to test a change on the corresponding pages, right ?
Then we should do the full change here and replace
img src="{{ STATIC_URL }}/img/gcd/flags/{{ profile_user.indexer.country.code|lower }}.png" alt="{{ country }}" class="embedded_flag" (with the <> around)
by
{{ profile_user.indexer.country|show_country_flag }}
to have the code the generate the flag path only in one place
you might need to add
{% load credits %}
in the beginning of a template if not already there

File: apps/indexer/templates/indexer/bits/profile_display.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
...
+  <li> Country: <img src="{{ STATIC_URL }}img/gcd/flags/{{ profile_user.indexer.country.code|lower }}.png" alt="{{ country }}" class="embedded_flag">
```
Comment (jochengcd) [inline apps/indexer/templates/indexer/bits/profile_display.html:17]: same comment

File: templates/gcd/admin/countries.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <h1>Countries used in series, creator and indexer tables</h1>
 
 {% for country in countries %}
-    <img src="{{ STATIC_URL }}/img/gcd/flags/{{ country.code|lower }}.png" alt=" " class="embedded_flag"> ({{ country.code|lower }}) {{ country }}<br/>
+    <img src="{{ STATIC_URL }}img/gcd/flags/{{ country.code|lower }}.png" alt=" " class="embedded_flag"> ({{ country.code|lower }}) {{ country }}<br/>
```
Comment (jochengcd) [inline templates/gcd/admin/countries.html:4]: same comment

File: templates/gcd/bits/series_issue_header.html [hunk 4]
```diff
@@ -45,7 +45,7 @@ <h1 class="item_id">
   {% endif %}
   {% if country %}
     ({{ country }})
-    <img src="{{ STATIC_URL }}/img/gcd/flags/{{ series.country.code|lower }}.png" alt="{{ country }}" class="embedded_flag">
+    <img src="{{ STATIC_URL }}img/gcd/flags/{{ series.country.code|lower }}.png" alt="{{ country }}" class="embedded_flag">
```
Comment (jochengcd) [inline templates/gcd/bits/series_issue_header.html:48]: same comment

File: templates/gcd/status/international_stats.html [hunk 5]
```diff
@@ -24,7 +24,7 @@
 <div class="stats">
         <h3 class="stats">
         <div class='stats_header'>
-            Statistics for Comics {% if type == 'country' %} published in <img style="vertical-align:middle" src="{{ STATIC_URL }}/img/gcd/flags/{{ object.code|lower }}.png" alt="{{ object }}">{% else %} in {% endif %} <a href="{% url "process_advanced_search" %}?target=series&method=icontains&logic=False&order1=series&order2=date&order3=&{{ type }}={{ object.code }}">{{ object }}{% if object.native_name %} ({{ object.native_name }}) {% endif %}</a>
+            Statistics for Comics {% if type == 'country' %} published in <img style="vertical-align:middle" src="{{ STATIC_URL }}img/gcd/flags/{{ object.code|lower }}.png" alt="{{ object }}">{% else %} in {% endif %} <a href="{% url "process_advanced_search" %}?target=series&method=icontains&logic=False&order1=series&order2=date&order3=&{{ type }}={{ object.code }}">{{ object }}{% if object.native_name %} ({{ object.native_name }}) {% endif %}</a>
```
Comment (jochengcd) [inline templates/gcd/status/international_stats.html:27]: here
src="{{ STATIC_URL }}img/gcd/flags/{{ object.code|lower }}.png" alt="{{ object }}"
by
{{ object|show_country_info_by_code }}

**Review by jochengcd — CHANGES_REQUESTED** [review]

**Comment by jochengcd** [comment]
If you don't want to do the code cleanup, this is fine as well. 
But, usually we should aim to cleanup code when doing changes at places. This is a case for a supporting routine implemented at some point, but not brought into play in all places were it could have been used.

**Pull request merged**